src/playback/internal/playbackcontroller.cpp

```diff
@@ -701,6 +711,77 @@ void PlaybackController::resume()
     currentPlayer()->resume(delay);
 }
 
+void PlaybackController::selectChordRestAtRawTick(const tick_t& rawTick)
```

Comment:
It would be better to move this method to interaction. The controller is essentially just a dispatcher, not an implementer.